### ISSUE TITLE
Creating a To-Do-List Using JavaScript

### ISSUE THREAD

**Issue by RAGHAVS1304** [Sar-taj107/Code-Sandbox-Hacktoberfest23#33]
## TechStack
- HTML
- CSS(Bootstrap)
-  Javascript

**Comment by RAGHAVS1304** [comment]
@Sar-taj107 Sir please assign this issue to me.
Thanks.

**Comment by Sar-taj107** [comment]
yeah sure @RAGHAVS1304  you can start but one thing keep in your mind that firstly you create a separate folder named "Projects" inside this folder you can put your project implementation with appropriate name that means in your case 'To-Do-List' and then make required changes in all your files for this folder and finally create a PR!
ex:
Projects/To-Do-List/your_files 

happy hcking^^
